packages/widgets/src/HeaderBar/TextIcon.js

```diff
@@ -1,16 +1,17 @@
 import propTypes from '@dhis2/prop-types'
 import React from 'react'
 
-export const TextIcon = ({ name, onClick, dataTestId }) => {
-    let title = name[0]
-
-    if (name.indexOf(' ') > 0) {
-        title += name.split(' ')[1][0]
-    }
+const getInitials = name =>
+    name
+        .split(' ')
+        .slice(0, 2)
+        .map(n => n[0])
+        .join('')
 
+export const TextIcon = ({ name, onClick, dataTestId }) => {
     return (
         <div onClick={onClick} data-test={dataTestId}>
-            <p>{title}</p>
+            <span>{getInitials(name)}</span>
```

Comment:
I haven't checked the layout, but are we intentionally going with a `span` as opposed to a `div`? `p` is a block level element, so if we want a semantically neutral element a `div` (also block level) seems more appropriate to me.

Reply:
I just went with `span` as that's what was suggested in the bug report but you're right that a `div` would make more sense due to being block level.

Reply:
Yeah. Maybe there's something we're not aware of. @amcgee, do you agree with the above, or is there something we're overlooking?